scripts/kind-with-registry.sh

```diff
@@ -19,9 +19,23 @@ set -o pipefail
 
 # Install kubectl and kind
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+# shellcheck source=hack/ensure-azcli.sh
+source "${REPO_ROOT}/hack/ensure-azcli.sh"
+# shellcheck source=hack/ensure-tags.sh
+source "${REPO_ROOT}/hack/ensure-tags.sh"
+
 KUBECTL="${REPO_ROOT}/hack/tools/bin/kubectl"
 KIND="${REPO_ROOT}/hack/tools/bin/kind"
-AZWI_ENABLED=${AZWI:-}
+AZWI="${REPO_ROOT}/hack/tools/bin/azwi"
+AZWI_ENABLED=${AZWI_ENABLED:-true}
+RAND_SUFFIX=$(openssl rand -hex 4)
+export AZWI_STORAGE_ACCOUNT="oidcissuer${RAND_SUFFIX}"
```

Comment:
Can you change this to not overwrite `AZWI_STORAGE_ACCOUNT` if it's already set?